spidermon/contrib/scrapy/monitors/monitors.py

```diff
@@ -528,6 +529,10 @@ class ZyteJobsComparisonMonitor(BaseStatMonitor):
     You can also filter which jobs to compare based on their tags using the
     ``SPIDERMON_JOBS_COMPARISON_TAGS`` setting. Among the defined tags we consider only those
     that are also present in the current job.
+
+    You can also filter which jobs to compare based on their close reason using the
+    ``SPIDERMON_JOBS_COMPARISON_CLOSE_REASONS`` setting. The default value is ``()``,
+    which doesn't filter any job based on close_reason.
```

Comment:
**suggestion:** I'd maybe add an example as I think most developers might want to set this.

Something like this:

 ```suggestion
    which doesn't filter any job based on close_reason. To only consider successfully finished jobs,
    use ``("finished", ) instead.``
```